clang/compiler-course/lysov_ivan_1stLabs/lysov_ivan_1stLabs.cpp

```diff
@@ -0,0 +1,89 @@
+#include "clang/AST/ASTConsumer.h"
+#include "clang/AST/Attr.h"
+#include "clang/AST/RecursiveASTVisitor.h"
+#include "clang/Frontend/CompilerInstance.h"
+#include "clang/Frontend/FrontendPluginRegistry.h"
+#include "clang/Rewrite/Core/Rewriter.h"
+#include "llvm/Support/raw_ostream.h"
+
+using llvm::outs;
+
+namespace {
```

Comment:
At the moment, you're just lucky that we're working with an anonymous namespace within a single file... Never use using-declaration in global scope :)
```suggestion
namespace {
using llvm::outs;
```